FusionIIIT/applications/alumni/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+# Create your models here.
+
+class Job(models.Model):
+    start_date = models.DateField()
+    end_date=models.DateField()
+    skill_required=models.TextField(max_length=500)
+    job_title = models.CharField(max_length=300)
+    job_description = models.TextField()
+    job_recommendedby = models.ForeignKey(Alumni,on_delete=models.CASCADE)
+    job_company = models.TextField()
+
+    class Meta:
+        db_table = 'Job'
+
+class Alumni(models.Model):
```

Comment:
Please Add more fields:
1. Past Experience
2. Present role and company

We plan to store this data in the database and thus require these fields

Reply:
Add more fields regrading relevant for an Alumni's profile.